### PR TITLE
Added PGN 65285 for Lowrance Temperature sensor

### PR DESCRIPTION
**Pull request by lfrancois**
First time commit adding a new PGN, so open to suggestions and edits.

### CODE REVIEW

**Comment by lfrancois** [comment]
```shell 
ludo@idefix:~/repos/forks/n2k-signalk % cat kbox-logs_2017-08-06_telnet_output.txt | ../../canboat/rel/darwin-x86_64/analyzer -json 2> /dev/null | grep 65285 | head -1 | ./bin/n2k-signalk
Illegal value in delta:{"value":25.6}
{"mmsi":"230099999"}%
```

**Comment by canboat** [comment]
Shouldn't that temperature source come from canboat/analyzer?

I'm sure that `source = 0` is what you see and that for you it is the engine room, but are you sure that this is the official name for this source, and if so, what other sources are there? Can you try and change your setup to other types and come up with a list please, then create an issue for https://github.com/canboat/canboat or even better a pull request. Thanks!

**Comment by lfrancois** [comment]
@canboat: yes the issue is in canboat, over the week-end I tried to switch the source to different values and canboat is always reporting `source = 0`. I will investigate more next week-end.

**Comment by tkurki** [comment]
Any developments on this?

**Comment by tkurki** [comment]
Travis CI complains about a syntax error:
```
/home/travis/build/SignalK/n2k-signalk/test/65285_temperature-2.js:2
  "65285-engineroom": {
                    ^
SyntaxError: Unexpected token :
```

**Comment by lfrancois** [comment]
@tkurki I updated the name of the data sample file with something different than js, to pass travis-ci checks.

Now we still have an issue with the 'Temperature Source' field, but it's `canboat` always returning the value 0.

**Comment by lfrancois** [comment]
Everything looks good now. But I guess I made a mistake when I wanted to pull the new commits from the master branch. What is the best way to fix it?

**Comment by lfrancois** [comment]
Hi @tkurki ! Can you review it please? And give me directions how to fix it or fix it for me :)

**Comment by tkurki** [comment]
Could you please rebase this on master?

**Comment by lfrancois** [comment]
@tkurki: done!

File: test/65285_temperature.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    var n2kMapper = require('../n2kMapper.js')
+    var full = new (require('@signalk/signalk-schema')).FullSignalK(
+      'urn:mrn:imo:mmsi:230099999'
+    )
+    _.forOwn(msgs, function (msg, key) {
+      var delta = n2kMapper.toDelta(msg)
+      delta.context = 'vessels.urn:mrn:imo:mmsi:230099999'
+      full.addDelta(delta)
+      delta.should.be.validSignalKDelta
+      delta.updates[0].values.forEach(function (pathValue) {})
```
Comment (tkurki) [inline test/65285_temperature.js]: This does not do anything?

File: test/65285_temperature.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+var msgs = require('./65285_temperature-2.json')
+
+describe('Temperature: ', function () {
+  it('examples work', function () {
+    var n2kMapper = require('../n2kMapper.js')
+    var full = new (require('@signalk/signalk-schema')).FullSignalK(
+      'urn:mrn:imo:mmsi:230099999'
+    )
+    _.forOwn(msgs, function (msg, key) {
```
Comment (tkurki) [inline test/65285_temperature.js]: Your test data file has only one sample, so the loop here does not do much. I would remove it and add something that tests the conversion result’s actual value. As it is the output could have a constant 0 value for example and the test would still be happy.

File: test/65285_temperature.js (hunk 1 above)
Comment (lfrancois) [reply test/65285_temperature.js]: Hmm I just tried to mock `13031_temperature.js`. Going to look at it again.

Comment (tkurki) [reply test/65285_temperature.js]: Yes, that test is shaped like that to test that the different 13031x samples in https://github.com/SignalK/n2k-signalk/blob/master/test/130310-2.json all result in valid SK output. Probably should add some test there that checks for correct output values as well.

**Comment by lfrancois** [comment]
@tkurki: updated, now I understand better the testing process.

**Pull request merged**